TaskForces/a11y/reports/audio-playback/index.html

```diff
@@ -119,52 +122,57 @@ <h2>Full Audio Publications</h2>
-				being able to render text content auditorily using text-to-speech synthesis. Even sighted readers will
+			<p>Text-To-Speech synthesis is commonly associated with users who are blind, but many readers benefit from
+				being able to render text content auditorily using Text-To-Speech synthesis. Even sighted readers will
 				turn to speech synthesis when it is not conducive to read content visually (e.g., when in a moving
-				vehicle).</p>
+				vehicle). Many Reading Systems are now providing TTS rendering as a function in the application. </p>
 
-			<p>Unlike audiobooks and publications with full audio, however, text-to-speech synthesis is not an audio
-				format. It is a way of listening to text content the reader has alread obtained. Readers can use a TTS
+			<p>Unlike audiobooks and publications with full audio, however, Text-To-Speech synthesis is not an audio
+				format. It is a way of listening to text content the reader has already obtained. Readers can use a TTS
 				application to read their EPUB publications, for example.</p>
 
-			<p>The applications that provide text-to-speech synthesis are usually not specifically designed for reading
-				publications, however. They are general tools that aid navigation across the device the reader is using
-				and any other apps and content that are on it. As a result, they often only have a very limited built-in
-				vocabulary of pronunciations so are not able to provide a similar quality of playback as prerecorded
-				human narration.</p>
+			<p>The applications that provide Text-To-Speech synthesis for persons with disabilities are usually not
+				specifically designed for reading publications, however. They are general tools that aid navigation
+				across the device the reader is using and any other apps and content that are on it. As a result, they
+				often only have a very limited built-in vocabulary of pronunciations so are not able to provide a
+				similar quality of playback as prerecorded human narration.</p>
 
-			<p>When a user enables text-to-speech playback, the text content of the publication is fed to an underlying
+			<p>When a user enables Text-To-Speech playback, the text content of the publication is fed to an underlying
 				TTS engine that voices each word. For most general language, the rendering returned is reasonably good,
 				but the engines will struggle with works that contain complex terms, jargon, uncommon names, etc.
 				Readers consequently also typically have the option to have individual words spelled out, both to help
-				disambiguate similar-sounding words and also to make sense of complex words and homonyms (words spelled
-				the same but with different pronunciations) that are mispronounced.</p>
+				disambiguate similar-sounding words and also to make sense of complex words and heteronyms (words
+				spelled the same but with different pronunciations) that are mispronounced. This ability explore the
```

Comment:
should there be a 'to' in this sentence: "This ability explore the spelling of words" > "This ability to explore the spelling of words"